cloud-config/run-scripts/app-pg.sh

```diff
@@ -68,13 +69,6 @@ if [ ! -f "$pg_keys_dir/credentials" ]; then
 fi
 
 ## Copy postgres aws to home
-pg_keys_dir='/home/ubuntu/pg-aws-keys'
```

Comment:
Not sure why this is deleted.

Reply:
It's repeated lines of code. Can make a separate branch, lmk.

Reply:
this is probably fixed in current cloud config.